packages/wasm/crate/src/planner.rs

```diff
@@ -101,7 +101,7 @@ impl ActionList {
         for value in self.balance().provided() {
             self.change_outputs.insert(
                 value.asset_id,
-                OutputPlan::new(&mut OsRng, value, change_address),
+                OutputPlan::new(&mut OsRng, value, change_address.clone()),
```

Comment:
For some reason, Rust was complaining about ownership issues here, despite there being no other changes to the planner.